### ISSUE TITLE
[juju 2.8-rc1] unknown facade CAASAdmission

### ISSUE THREAD

**Issue by VariableDeclared** [juju/python-libjuju#409]
Hi, 

The facade CAASAdmission is missing from `juju/client/connection.py client_facades` dictionary  resulting in key error at runtime:

```
________________________________________________________________ ERROR at setup of test_rbac _________________________________________________________________
Traceback (most recent call last):                                                                                                                            
  File "/home/ubuntu/k8s-validation/py3/lib/python3.6/site-packages/pytest_asyncio/plugin.py", line 102, in wrapper                                           
    return asyncio.get_event_loop().run_until_complete(setup())                                                                                               
  File "/usr/lib/python3.6/asyncio/base_events.py", line 484, in run_until_complete                                                                           
    return future.result()                                                                                                                                    
  File "/home/ubuntu/k8s-validation/py3/lib/python3.6/site-packages/pytest_asyncio/plugin.py", line 85, in setup                                              
    res = await gen_obj.__anext__()                                                                                                                           
  File "/home/ubuntu/k8s-validation/jobs/integration/conftest.py", line 138, in model                                                                         
    await model.connect(tools.connection)                                                                                                                     
  File "/home/ubuntu/k8s-validation/py3/lib/python3.6/site-packages/juju/model.py", line 537, in connect                                                      
    await self._connector.connect_model(model_name, **kwargs)                                                                                                 
  File "/home/ubuntu/k8s-validation/py3/lib/python3.6/site-packages/juju/client/connector.py", line 143, in connect_model                                     
    bakery_client=self.bakery_client_for_controller(controller_name),                                                                                         
  File "/home/ubuntu/k8s-validation/py3/lib/python3.6/site-packages/juju/client/connector.py", line 69, in connect                                            
    self._connection = await Connection.connect(**kwargs)                                                                                                     
  File "/home/ubuntu/k8s-validation/py3/lib/python3.6/site-packages/juju/client/connection.py", line 299, in connect                                          
    await self._connect_with_redirect([(endpoint, cacert)])                                                                                                   
  File "/home/ubuntu/k8s-validation/py3/lib/python3.6/site-packages/juju/client/connection.py", line 671, in _connect_with_redirect                           
    self._build_facades(login_result.get('facades', {}))                                                                                                      
  File "/home/ubuntu/k8s-validation/py3/lib/python3.6/site-packages/juju/client/connection.py", line 694, in _build_facades                                   
    known = client_facades[name]['versions']                                                                                                                  
KeyError: 'CAASAdmission'                                                                                                                                     
--------------------------------------------------------------------- Captured log setup --------------------------------------------------------------------$                                                                                                           
```

Many thanks for your help
Peter

**Comment by hpidcock** [comment]
This issue should be solved by https://github.com/juju/python-libjuju/pull/382

**Comment by hpidcock** [comment]
@VariableDeclared do you know what version of python-libjuju you are using?

**Comment by VariableDeclared** [comment]
hey @hpidcock thanks for checking that,

I'm using the latest from the package index: https://pypi.org/project/juju/ aka 2.7.1. I can try a manual install of though just to confirm.

Cheers

**Comment by VariableDeclared** [comment]
@hpidcock can confirm that it is fixed by #382. Can we get this into the python package index?

**Comment by hpidcock** [comment]
https://pypi.org/project/juju/2.8.0/ has been released
